Index: .idea/webContexts.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/webContexts.xml b/.idea/webContexts.xml
new file mode 100644
--- /dev/null	(date 1713902321636)
+++ b/.idea/webContexts.xml	(date 1713902321636)
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="WebContextManager">
+    <option name="state">
+      <map>
+        <entry key="file://$PROJECT_DIR$/src/main/webapp/WEB-INF/jsp/view/base.jsp" value="file://$PROJECT_DIR$/src/main/webapp/WEB-INF/jsp/view" />
+        <entry key="file://$PROJECT_DIR$/src/main/webapp/WEB-INF/jsp/view/listTickets.jsp" value="file://$PROJECT_DIR$/src/main/webapp/WEB-INF/jsp/view" />
+        <entry key="file://$PROJECT_DIR$/src/main/webapp/WEB-INF/jsp/view/login.jsp" value="file://$PROJECT_DIR$/src/main/webapp/WEB-INF/jsp/view" />
+        <entry key="file://$PROJECT_DIR$/src/main/webapp/WEB-INF/jsp/view/ticketForm.jsp" value="file://$PROJECT_DIR$/src/main/webapp/WEB-INF/jsp/view" />
+        <entry key="file://$PROJECT_DIR$/src/main/webapp/WEB-INF/jsp/view/viewTicket.jsp" value="file://$PROJECT_DIR$/src/main/webapp/WEB-INF/jsp/view" />
+        <entry key="file://$PROJECT_DIR$/src/main/webapp/index.jsp" value="file://$PROJECT_DIR$/src/main/webapp" />
+      </map>
+    </option>
+  </component>
+</project>
\ No newline at end of file
Index: .idea/.name
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.name b/.idea/.name
new file mode 100644
--- /dev/null	(date 1713902321648)
+++ b/.idea/.name	(date 1713902321648)
@@ -0,0 +1,1 @@
+Attachment.java
\ No newline at end of file
Index: src/main/java/com/example/crbyerscustomersupport/Attachment.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>// NAME: <Chelsie Byers>\r\n// CLASS: Info 1541/Spring\r\n// ASSIGNMENT: <Crbyers-Assignment 4>\r\n// DATE: <3/27/2024/4/3/2024>\r\n// RESOURCES: <I used the resource videos and lectures from this class and Professional Java for Web Applications>\r\n\r\n//This program will allow a user to input a ticket to report any support issues with their accounts>\r\n\r\npackage com.example.crbyerscustomersupport;\r\n\r\npublic class Attachment {\r\n    //set up my variables\r\n\r\n    private static String name;\r\n\r\n    private static byte[] contents;\r\n\r\n    //set up my getters and setters\r\n    public static String getName() {\r\n        return name;\r\n    }\r\n\r\n    public void setName(String name) {\r\n        this.name = name;\r\n    }\r\n\r\n    public static byte[] getContents() {\r\n        return contents;\r\n    }\r\n    //set up my java bean contents\r\n\r\n    public void setContents(byte[] contents) {\r\n        this.contents = contents;\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/crbyerscustomersupport/Attachment.java b/src/main/java/com/example/crbyerscustomersupport/Attachment.java
--- a/src/main/java/com/example/crbyerscustomersupport/Attachment.java	(revision 7beda79655b67c7ed6880955c75b7504e1ad812a)
+++ b/src/main/java/com/example/crbyerscustomersupport/Attachment.java	(date 1713902321623)
@@ -15,6 +15,7 @@
 
     private static byte[] contents;
 
+
     //set up my getters and setters
     public static String getName() {
         return name;
